src/vegur_utils.erl

```diff
@@ -253,6 +253,10 @@ connection_info(Items, Req) ->
             ProxySocket = cowboyku_req:get(socket, Req),
             {ok, Ret} = Transport:connection_info(ProxySocket, Items),
             {Ret, Req};
+        proxy_protocol_ssl ->
+            ProxySocket = cowboyku_req:get(socket, Req),
+            {ok, Ret} = Transport:connection_info(ProxySocket, Items),
+            {Ret, Req};
```

Comment:
Well it felt like it was minimal duplication to avoid going
```erlang
Name = Transport:name(),
if Name =:= proxy_protocol_ssl; Name =:= proxy_protocol_tcp ->
       ProxySocket = cowboyku_req:get(socket, Req),
      {ok, Ret} = Transport:connection_info(ProxySocket, Items),
      {Ret, Req};
    true ->
     {[], Req}
end
```

But if that's easier to read, I can fix that up